pwiz_tools/Shared/Common/Collections/CollectionUtil.cs

```diff
@@ -202,7 +202,7 @@ public static int CompareColumnValues(object o1, object o2)
             {
                 return 1;
             }
-            return Comparer.Default.Compare(o1.ToString(), o2.ToString());
+            return NaturalComparer.Compare(o1.ToString(), o2.ToString()); // Compare using natural non-lexicographical order for easier human readability (1,20,3 --> 1,3,20)

```

Comment:
Closer - but the point is that strings of mixed letters and numbers sort a certain way. Just 
(a1b,a20b,a3b --> a1b,a3b,a20b)
would be an improvement in clarity